src/probnum/filtsmooth/statespace/integrator.py

```diff
@@ -127,33 +136,26 @@ def transition_rv(self, rv, start, stop, _diffusion=1.0, **kwargs):
             raise TypeError(errormsg)
         step = stop - start
         rv = self.precon.inverse(step) @ rv
-        rv, info = self.transition_rv_preconditioned(rv, start, _diffusion=_diffusion)
-        info["crosscov"] = self.precon(step) @ info["crosscov"] @ self.precon(step).T
-        return self.precon(step) @ rv, info
-
-    def transition_rv_preconditioned(self, rv, start, _diffusion=1.0, **kwargs):
-        return self.equivalent_discretisation_preconditioned.transition_rv(
+        rv, info = self.equivalent_discretisation_preconditioned.transition_rv(
             rv, start, _diffusion=_diffusion
         )
+        info["crosscov"] = self.precon(step) @ info["crosscov"] @ self.precon(step).T
+        return self.precon(step) @ rv, info
 
     def transition_realization(self, real, start, stop, _diffusion=1.0, **kwargs):
         if not isinstance(real, np.ndarray):
             raise TypeError(f"Numpy array expected, {type(real)} received.")
         step = stop - start
         real = self.precon.inverse(step) @ real
-        real, info = self.transition_realization_preconditioned(
+        (
+            real,
+            info,
```

Comment:
Ok this seems to be a case where Black does really not lead to nice code :see_no_evil:

Reply:
Obviously this does is not meant to be a criticism of your code style^^ I don't see how else to do things

Reply:
I agree... but hey, black cannot do everything perfectly! I changed the code here to `out = ...; real, info = out` and black is fine with that now.